README.md

```diff
@@ -1,2 +1,4 @@
-# af_colours
-Python package to produce the Analysis Function recommended colour palettes.
+# py_af_colours
+This page contains the documentation for the Python package of the Analysis Function’s colour palettes. The py_af_colours package aims to maintain the accessibility of graph outputs generated in Python across the Analysis Function by simplifying the process by which users can access the standardised colour palettes.
```

Comment:
This sentence is a good opener for the documentation page but in the readme it should be about the package as a whole. I would recommend slightly rephrasing it - [govcookiecutter](https://github.com/best-practice-and-impact/govcookiecutter/blob/main/README.md) is a nice example